parking_permits/models/driving_licence.py

```diff
@@ -20,7 +21,7 @@ class DrivingLicence(TimestampedModelMixin, UUIDPrimaryKeyMixin):
     active = models.BooleanField(null=False, default=True)
 
     def is_valid_for_vehicle(self, vehicle):
-        is_not_expired = self.valid_end > arrow.utcnow()
+        is_not_expired = self.valid_end > datetime.date.today()
```

Comment:
I think we should always use local time for comparison. `datetime.date.today()` will return a server local time, which might not be using a Europe/Helsinki time. To ensure that we are using the local time, we could do something like (this will return current date in TIME_ZONE settings):
```suggestion
        is_not_expired = self.valid_end > timezone.localdate(timezone.now())
```

Reply:
Good catch. Updated now.